modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/50_dsl.js

```diff
@@ -8,6 +8,33 @@ var callLuciDSLStatus = rpc.declare({
 	expect: { '': {} }
 });
 
+function formatElapsedTime(seconds) {
+	var d = seconds / 86400;
+	seconds = seconds % 86400;
+	var h = seconds / 3600;
+	seconds = seconds % 3600;
+	var m = seconds / 60;
+	seconds = seconds % 60;
+
+	if (d > 0)
+		return '%dd %dh %dm %ds'.format(d, h, m, seconds);
+	if (h > 0)
+		return '%dh %dm %ds'.format(h, m, seconds);
+	if (m > 0)
+		return '%dm %ds'.format(m, seconds);
+	if (s > 0)
+		return '%ds'.format(seconds);
+	return null;
+}
```

Comment:
I think you can simply use the [`%t` format](https://github.com/openwrt/luci/blob/master/modules/luci-base/htdocs/luci-static/resources/cbi.js#L611) here to shrink the code.

Reply:
Done